File: esperanza/0ther/silver/python/xag.py
from metalpriceapi.client import Client
import json

api_key = ''
client = Client(api_key)

res = client.fetchLive(base='XAG', currencies=['EUR', 'USD', 'CNY', 'RUB'])

json_data = json.dumps(res)

with open('/home/silver/python/result.txt', 'w') as f:
    f.write(str(json_data))

File: esperanza/0ther/silver/python/xag.ts
import api from "npm:metalpriceapi"

api.setAPIKey('')
const res = await api.fetchLive('XAG', ['EUR', 'USD', 'CNY', 'RUB'])
const dataToSave = JSON.stringify(res.data, null, 2)
await Deno.writeTextFile('/home/silver/python/result.txt', dataToSave)
